.github/workflows/master.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         run: |
           TEST_MODULES="dev/kyuubi-codecov"
           ./build/mvn clean install ${MVN_OPT} -pl ${TEST_MODULES} -am \
-          -Pspark-${{ matrix.spark }} ${{ matrix.spark-archive }} ${{ matrix.exclude-tags }}
+          -Pspark-${{ matrix.spark }} -Phudi ${{ matrix.spark-archive }} ${{ matrix.exclude-tags }}
```

Comment:
Any possible way to prevent enable a Hudi specific profile for default compilation on Scala 2.12 here? Or shall we could do the proper exclusion for  Scala 2.13?

Reply:
Try some method but not work.